.github/linters/ruff.toml

```diff
@@ -39,8 +39,8 @@ target-version = "py38"
 # Enable Pyflakes (`F`) and a subset of the pycodestyle (`E`)  codes by default.
 # Unlike Flake8, Ruff doesn't enable pycodestyle warnings (`W`) or
 # McCabe complexity (`C901`) by default.
-select = ["E4", "E7", "E9", "F"]
-ignore = ["E721", "E722", "E731", "F401", "F402", "F403", "F405", "F811", "F821", "F822", "F841", "F901"]
+select = ["E3", "E4", "E5", "E7", "E9", "F"]
+ignore = ["E501", "E721", "E722", "E731", "F401", "F402", "F403", "F405", "F811", "F821", "F822", "F841", "F901"]
```

Comment:
We are using black as the python linter. Will black use lint configs defined in `ruff.toml`?

Reply:
I have removed ruff now.  Thanks